cisagov/manage.get.gov
# Issue #2055 - registrar user contact info preamble

## Ticket 2055

Resolves #2055


## Changes


- Added a summary box with user's profile information to Preamble, utilizing the Profile waffle flag so the region only shows if that flag is enabled.




## Context for reviewers



## Setup



## Code Review Verification Steps
Code is deployed to [sandbox-bob](https://getgov-bob.app.cloud.gov/)

- Go to admin and enable the Profile waffle flag for all users. 
- [ ] Start a new domain request and verify that the contact information section appears in the preamble as shown below.
- Go to admin and disable the Profile waffle flag.
- [ ] Start a new domain request and verify that the contact information section DOES NOT appear in the preamble.


<img width="725" alt="image" src="https://github.com/cisagov/manage.get.gov/assets/16280277/ee718372-8815-4d39-8181-fb800fb953d8">


### As the original developer, I have

#### Satisfied acceptance criteria and met development standards


#### Ensured code standards are met (Original Developer)


#### Validated user-facing changes (if applicable)


### As a code reviewer, I have

#### Reviewed, tested, and left feedback about the changes


#### Ensured code standards are met (Code reviewer)


#### Validated user-facing changes as a developer




**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

### As a designer reviewer, I have

#### Verified that the changes match the design intention


#### Validated user-facing changes as a designer




## Screenshots
